source/Account.cpp

```diff
@@ -136,6 +136,9 @@ string Account::Step(int64_t assets, int64_t salaries, int64_t maintenance)
 	maintenanceDue += maintenance;
 	bool missedPayment = false;
 
+	credits -= escortCosts;
+	out << "You paid " << escortCosts << " credits for escorts. ";
```

Comment:
Won't this leave a message even if escortCosts is equal to 0?

Reply:
Yes, you  are right I just didnt thought of an escort that has no cost